docs/spec_draft.md

```diff
@@ -1327,6 +1328,42 @@ behavior is undefined. More formally, for all `id < jd` from `indices(result)`,
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.get_tuple_element
+
+### Semantics
+
+Returns the element at `index` position of the `operand` tuple as `result`.
+
+### Inputs
+
+| Name      | Type                                                                       |
+|-----------|----------------------------------------------------------------------------|
+| `operand` | a variadic sized tuple of tokens, tuples, or tensors of any supported type |
+| `index`   | constant of type `ui32`                                                    |
+
+### Outputs
+
+| Name     | Type                                          |
+|----------|-----------------------------------------------|
+| `result` | token, tuple, or tensor of any supported type |
+
+### Constraints
+
+  * (C1) 0 $\le$ `index` $\lt$ size(`operand`).
```

Comment:
Verifier doesn't check that `index >= 0`.

Reply:
it does not, actually for ui32 this is not required.

Reply:
I think I had a similar issue in a previous PR related to how it is currently signless and bitcasted to uint32, but I forget which PR it was. Let's keep this open for now.

Reply:
Yes, let's use `si32` here consistently with the attributes specced so far and figure out signedness as part of #481. Also let's update C1 accordingly. It's fine to keep "Verification" at "yes" given that the verifier is written for signless ints.